packages/doctype-caip10-link-handler/src/caip10-link-doctype-handler.ts

```diff
@@ -87,6 +81,16 @@ export class Caip10LinkDoctypeHandler implements DoctypeHandler<Caip10LinkDoctyp
         return state
     }
 
+    /**
+     * Thin wrapper around 'validateLink' function from blockchain-utils-validation to allow
+     * injecting a mock implementation for ease of unit testing.
+     * @param proof
+     * @private
+     */
```

Comment:
I guess my concern with that would be that I'd have to mock the whole module for all consumers, rather than just for a specific Caip10Link instance at a time.  But for the test as I have it now it doesn't really matter.  Mostly I just didn't have a good example of how to do that and didn't want to spend too long figuring it out, but I can look into it today.  It does seem kind of gross and silly to add this just as a crutch for testing